### PR TITLE
Bar & Scatter charts for individual columns

### PR DESCRIPTION
**Pull request by breville**
This moves our bar & scatter plot charts to use Chart.js.  

A scatter plot is now shown for a continuous feature vs. a continuous label, on the X & Y axes respectively.  

### Single continuous column:
Screen Shot 2021-03-03 at 4 13 31 PM

### Continuous feature & continuous label:
Screen Shot 2021-03-03 at 4 20 19 PM

### Single categorical column:
Screen Shot 2021-03-03 at 4 13 57 PM

### CODE REVIEW

**Comment by Erin007** [comment]
Is this still an active branch? Screenshots?

**Comment by breville** [comment]
@Erin007 Good question.  It was paused until the scatter plot work was ready to go.  That's now done, and it should be ready for review.

File: src/UIComponents/ColumnInspector.jsx [hunk 1]
```diff
@@ -12,7 +12,36 @@ import {
...
+    }
+  ]
+};
+
+const chartOptions = {
+  scales: {
+    yAxes: [
+      {
+        ticks: {
+          beginAtZero: true
```
Comment (Erin007) [inline src/UIComponents/ColumnInspector.jsx:37]: yay! This will make it easier for middle schoolers to read these graphs :)

**Review by Erin007 — APPROVED** [review]
LGTM!

**Pull request merged**